### DISCUSSION TITLE
需要帮助

### DISCUSSION THREAD

**Discussion by shaula1973** [HerbertHe/iptv-sources#85] [Q&A]
root@DS918:~# docker run -v /volume1/docker/iptv-sources/config:/app/config iptv-sources
Unable to find image 'iptv-sources:latest' locally
docker: Error response from daemon: pull access denied for iptv-sources, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.

why?

**Comment by shaula1973** [comment]
通过群晖的container manager在容器里添加目录映射，已经成功看到容器内部/config下面有了custom.json。但是执行docker exec -d iptv-sources /bin/sh ./update-sources.sh 后，查看fmml_ipv6.txt没有变化，是json有问题吗？内容如下：
{
    "rules": [
        {
            "upstream": "fmml_ipv6",
            "exclude": [],
            "include": [],
            "append": [],
            "replacer": {
              "extinf": <same as url>,
              "url": [
                    {
                        "pattern": "]/",
                        "type": "string",
                        "target": "]:80/"
                    }
                ]
            }
        }
    ]
}

**Comment by HerbertHe** [comment]
你的启动都是错的，镜像的名称是 herberthe0229/iptv-sources。

其次，自定义化之后的文件在 /custom/xx

**Comment by shaula1973** [comment]
1、 docker run -v启动提示错误，已经通过container manager解决config目录绑定本地问题；
2、已经按说明新建了custom.json，并且进入容器后能找到这个文件，说明第一步的目的已经达到；
3、容器里没发现/custom目录，不清楚json有没有写错还是根本没执行；
4、我以为自定义后会替换原来的输出 http://,自定义域名:端口/fmml_ipv6.m3u 文件。

**Reply by HerbertHe** [reply]
@shaula1973 按照 GitHub issue 按规范提交可复现步骤，json 文件内容通过代码块要规范化
